velox/exec/prefixsort/tests/utils/EncoderTestUtils.cpp

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "velox/exec/prefixsort/tests/utils/EncoderTestUtils.h"
+
+namespace facebook::velox::exec::prefixsort::test {
+
+namespace {
+/// The decode method will only be used in testing scenarios. In production
+/// code, we have no scenarios that require decoding a normalized key.
+void decodeNoNulls(bool ascending, char* encoded, int64_t& value) {
```

Comment:
'ascending' parameter is always false; hence, it is redundant. Let's remove it.

Reply:
done